docs/en/sql-reference/functions/type-conversion-functions.md

```diff
@@ -7211,6 +7211,40 @@ Result:
 └─────────────────────────────┘
 ```
 
+## toTypeName
```

Comment:
There is already documentation for this function: https://clickhouse.com/docs/en/sql-reference/functions/other-functions#totypename, but maybe you can improve it?

`type-conversion-functions.md` -- is definitely the wrong place for it. Let's keep it in `other-functions`

Reply:
Thanks for your reply. I will close this and improve the file that you mention :).